### PR TITLE
Adds getCurrentTime and getDuration methods to query video time values

### PR DESCRIPTION
**Pull request by owenworley**
Adds getCurrentTime and getDuration methods to query video time values.
Adds new buttons to demo application to demonstrate functionality.

### CODE REVIEW

File: YouTubePlayer/YouTubePlayer/YouTubePlayer.swift [hunk 1]
```diff
@@ -181,8 +189,12 @@ public class YouTubePlayerView: UIView, UIWebViewDelegate {
     }
 
     private func evaluatePlayerCommand(command: String) {
+        queryPlayerCommand(command)
+    }
+    
+    private func queryPlayerCommand(command: String) -> String? {
```
Comment (gilesvangruisen) [inline YouTubePlayer/YouTubePlayer/YouTubePlayer.swift]: How do you feel about rolling these back into one command and simply ignoring the return value for things like play/pause/etc.?

**Comment by gilesvangruisen** [comment]
This is an awesome change. Thanks @owenworley! Just one comment then I'll get this merged ASAP

File: YouTubePlayer/YouTubePlayer/YouTubePlayer.swift (hunk 1 above)
Comment (owenworley) [inline YouTubePlayer/YouTubePlayer/YouTubePlayer.swift]: I can do that, it seemed easier to reason about and more explicit to have one place that ignores the return value.

Commit incoming

**Comment by AlessandroTolomio** [comment]
Hi,
I've installed Swift-YouTube-Player with cocoa but there's not this feature. It's possible?

**Pull request merged**